css/templates/gallery/components/_landing-images.css

```diff
@@ -0,0 +1,570 @@
+   */
+  font-size: 13vw;
+  pointer-events: none;
+}
+
+@media (--breakpoint-sm) {
+  .gallery-landing-title {
+    font-size: 7vw;
+    padding: 1rem 5vw;
+  }
+}
+
+.gallery-landing-wall .amp-carousel-button {
+  background-color: white;
+  box-shadow: -10px 0 20px rgba(0,0,0,0.12);
+  border-radius: 0;
+  width: 50px;
+  height: 89px;
+  cursor: pointer;
+  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+}
+
+.gallery-landing-wall .amp-carousel-button:hover {
+  background-color: yellow;
+}
+
+.gallery-landing-wall .amp-carousel-button:focus {
+  outline: none;
+}
+
+.gallery-landing-wall .amp-carousel-button-next {
+  background-image: url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' width='18' height='18' fill='%23000' viewBox='0 0 18 18'%3E%3Cpath d='M9 3L7.94 4.06l4.19 4.19H3v1.5h9.13l-4.19 4.19L9 15l6-6z'/%3E%3C/svg%3E");
+  right: 0;
+}
+.gallery-landing-wall .amp-carousel-button-prev {
+  left: 0;
+  background-image: url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' width='18' height='18' fill='%23000' viewBox='0 0 18 18'%3E%3Cpath d='M15 8.25H5.87l4.19-4.19L9 3 3 9l6 6 1.06-1.06-4.19-4.19H15v-1.5z'/%3E%3C/svg%3E");
+}
+
+@media (max-width: 40.06rem) {
+  .gallery-landing-wall .amp-carousel-button {
+    position: fixed;
+    top: auto;
+    bottom: 0;
+    transform: none;
+    width: 55px;
+    height: 55px;
+  }
+}
+
+.gallery-landing-wall-scroll {
+  width: 285vh;
+  height: 100vh;
+  min-width: 100vw;
+  overflow: hidden;
+}
+
+@media (max-width: 40.06rem) {
+  .gallery-landing-wall-scroll {
+    margin-bottom: -3px; /* remove spacing from display inline-block */
+  }
+}
+
+@keyframes hover-one {
+  0% {transform: translateY(-2vh) rotateZ(360deg); }
+  50% {transform: translateY(2vh) rotateZ(360deg); }
+  100% {transform: translateY(-2vh) rotateZ(360deg); }
+}
+
+@keyframes hover-two {
+  0% {transform: translateY(-1vh) rotateZ(360deg); }
+  50% {transform: translateY(1vh) rotateZ(360deg); }
+  100% {transform: translateY(-1vh) rotateZ(360deg); }
+}
+
+[class^="gallery-landing-image-wrap-"] {
+  width: 25vh;
+  position: absolute;
+  display: block;
+  transform: translateZ(0);
+  transform-origin: 0px 0px 0px;
+
+  animation-name: hover-one;
+  animation-duration: 20s;
+  animation-iteration-count: infinite;
+  animation-timing-function: ease-in-out;
+  animation-fill-mode: both;
+
+  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+}
+
+.amp-mode-mouse .gallery-landing-image-link {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  z-index: 2;
+}
+
+[class^="gallery-landing-image-wrap-"]:nth-child(odd) {
+  animation-name: hover-two;
+}
+
+[class^="gallery-landing-image-wrap-"] amp-img {
+  object-fit: cover;
+  box-shadow: 1vh 6vh 5vh var(--color-shadow);
+  transition: transform 300ms;
+  pointer-events: none;
+}
+
+.gallery-landing-image-wrap-0 amp-img { background: #b3afab }
```

Comment:
I believe this was for selector performance -- selecting a single class is (very roughly) 50 to 100 milliseconds faster than an nth-child selector (as discovered using [this test](https://benfrain.com/selector-test/index.html). Since we have 18 of these, we end up saving 1 to 2 seconds on rendering just from this change. I can change it to use nth child if you'd prefer.